submissions/edehJohn/html/index.html

```diff
@@ -0,0 +1,97 @@
+<!Doctype html>
```

Comment:
Well done!
Why are you deleting your .txt file?
You did not add the images used in this file to this pull request?
I noticed you used more than 2 product images. You need to update your file @john84edeh

Reply:
thank you ma. I dont remember deleting any txt file, and sorry, what i my suppose to update  on the file ?,. to down size the product images or add the images used in the PR? or both?

Reply:
You are using images on your html file but the images are not added to the pull request.

Reply:
Make sure you are using ONLY 2 product images. @john84edeh